base-template/about-base-template.md

```diff
@@ -1,45 +1,63 @@
+To make the best use of this base template, you need a working knowledge of:
 
-### About the "Overview" section
+- GitHub to download, edit, and use the base template.
+- Markdown to create and edit the base template to suit your requirements.
+- Different documentation types to identify if you need to create a new template or edit an existing one.
+- Technical writing to produce a well-written document that includes the relevant reference aspects.
 
-Professionally redefine interactive deliverables whereas client-based e-commerce. Uniquely impact competitive quality vectors after resource-leveling content. Competently repurpose economically sound value before strategic processes. Competently benchmark global action items rather than user friendly internal or "organic" sources. Continually plagiarize flexible e-business after strategic core competencies.
+## Who will need the base template
 
-### About the content
+The base template is for:
 
-Energistically embrace worldwide applications without customized growth strategies. Energistically embrace orthogonal convergence vis-a-vis global internal or "organic" sources. Dramatically transform leading-edge networks rather than client-based convergence. Credibly revolutionize web-enabled interfaces before strategic solutions. Authoritatively incubate economically sound e-business with equity invested outsourcing.
+- Internal users - those who are already working in the Good Docs Project (GDP)
+- External users - the community outside of the GDP
 
-### About the "Reference" section
+Who want to:
 
-Professionally cultivate emerging alignments through multimedia based initiatives. Dramatically administrate cross-media users rather than cooperative paradigms. Synergistically redefine long-term high-impact niches via cross functional relationships. Authoritatively whiteboard interactive "outside the box" thinking with dynamic supply chains. Assertively pontificate standardized applications before goal-oriented sources.
+- Create new templates that do not yet exist within GDP
+- Edit existing templates that require changes
 
-### About the "Examples" section
+## How do you know you need the base template
 
-Synergistically redefine fully tested "outside the box" thinking before excellent customer service. Completely predominate functional methodologies vis-a-vis corporate core competencies. Energistically underwhelm dynamic supply chains through client-centric materials. Assertively negotiate client-based deliverables after timely infrastructures. Competently productize wireless ROI without customized mindshare.
+The base template is useful when:
 
-### About the "Glossary" section
+- You have identified that the process or document type you are trying to create does not have an existing template in GDP
```

Comment:
For most/all your lists in the doc, I feel you should have a period at the end.
Google's style guide is ambiguous in their requirements here, but it is fairly clear in Microsoft's guide:
https://docs.microsoft.com/en-us/style-guide/scannable-content/lists#punctuation
In bulleted and numbered lists, end each list item with a period if:
* Any item forms a complete sentence when combined with the list introduction that precedes the colon. ...

Reply:
Noted, and shall be corrected. Thanks for Google style guide reference.